ember-cli-build.js

```diff
@@ -3,12 +3,6 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
 
 module.exports = function (defaults) {
   let app = new EmberAddon(defaults, {
-    babel: {
-      optional: ['es7.decorators']
```

Comment:
Don't still need this?  I have not been checking for this entry in the other repos we've been updating.

Reply:
It is needed for `ember-computed-decorators` which is not being used in this add-on.